com.unity.render-pipelines.high-definition/Documentation~/Decal.md

```diff
@@ -36,9 +36,13 @@ After you enable Decal Layers, you can then use them to decouple Meshes from cer
 
 By default, in the UI for Decal Projectors, Mesh Renderers, or Terrain, Decal Layers are named **Decal Layer 1-7**. To more easily differentiate between them, you can give each Decal Layer a specific name. To do this, open the [HDRP Global Settings](Default-Settings-Window.md), and go to **Decal Layer Names**. Here you can set the name of each Decal Layer individually.
 
-### Performance implications
+### How decals affect performance
```

Comment:
I like how you've changed this title, but this section is specifically about Decal Layers. Replace "decals" with "Delal Layers" and we're good :)